jkube-kit/config/image/src/test/java/org/eclipse/jkube/kit/config/image/build/CleanupModeTest.java

```diff
@@ -41,20 +41,19 @@ public void parse() {
         };
 
         for (int i = 0; i < data.length; i += 2) {
-            assertEquals(data[i+1], CleanupMode.parse((String) data[i]));
+            assertThat(CleanupMode.parse((String) data[i])).isEqualTo(data[i + 1]);
         }
     }
 
     @Test
-    public void invalid() {
+    void invalid() {
         try {
             CleanupMode.parse("blub");
             fail();
         } catch (IllegalArgumentException exp) {
-            assertTrue(exp.getMessage().contains("blub"));
-            assertTrue(exp.getMessage().contains("try"));
-            assertTrue(exp.getMessage().contains("none"));
-            assertTrue(exp.getMessage().contains("remove"));
+            assertThatIllegalArgumentException()
+                    .isThrownBy(() -> CleanupMode.parse("blub"))
+                    .withMessageContainingAll("blub", "try", "none", "remove");
```

Comment:
I'm sorry for not being clear. I meant to replace whole try-catch block with this